common/templates/deployment.yaml

```diff
@@ -66,6 +66,10 @@ spec:
               value: {{ .Values.environment }}
             - name: DEPLOYMENT_NAME
               value: {{ include "common.name" . }}
+           {{- range $name, $value := .Values.env }}
```

Comment:
We already mount these env variables with a configmap, https://github.com/dave-inc/charts/blob/a109d91ea07f240c2ea063b3a9f535320aa98dd2/common/templates/deployment.yaml#L73-L75

configmap here, https://github.com/dave-inc/charts/blob/a109d91ea07f240c2ea063b3a9f535320aa98dd2/common/templates/configmap.yaml